README.md

```diff
@@ -0,0 +1,56 @@
+<p align="center">
```

Comment:
Loved the explanation, I just don't feel the "Meilisearch Javascript plugins" is the right choice, but I don't have any other suggestion, so I'm just thinking out loud!

Maybe @alallema has one?

Reply:
After internal discussion, the repo will be named `meilisearch-js-plugins`. Thus the title may be kept as `javascript` inside the readme